src/styles/base/footer.scss

```diff
@@ -1,12 +1,22 @@
 .footer {
     padding: 40px 0;
+    font-size: 14px;
+    line-height: urset;
 
     &__link {
         color: $color-blue-text;
         font-weight: 500;
 
         &--light {
             color: $color-gray-text;
+
+            &:hover {
```

Comment:
`link--light` has the same hover as `link`, so no need to specify it once more.
Remove `&--light &:hover` block please.